pkg/images/images.go

```diff
@@ -0,0 +1,336 @@
+package images
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"text/template"
+
+	"github.com/blang/semver"
+	utiliies "github.com/rancher/kontainer-driver-metadata/pkg"
+	"github.com/rancher/norman/types/convert"
```

Comment:
It seems like you are only using `convert.ToObj` function in `norman` package. It would be easier to maintain with your own `ToObj` function and remove `norman` package imported.

The package `norman` has a bunch of k8s deps with specific version which might lead to deps conflict.

Reply:
norman is dropped as you suggested 😄